docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb

```diff
@@ -0,0 +1,425 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    username = maap.profile.account_info()["username"]
</pre><p>Is there a reason to print your username?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>I would just remove this line altogether</p>

Reply:
<p>This has not been fixed yet.</p>